examples/issue_789/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
```

Comment:
examples should run with the current project requirements file. Can you please delete these?

Reply:
Sure, I will do.

Reply:
I have done that and update the README as well.